### PR TITLE
Page: Use `Sound::SoundId` for `setAnimSfxIds` parameters

### PR DESCRIPTION
**Pull request by riidefi**

### CODE REVIEW

File: payload/game/ui/Page.hh [hunk 1]
```diff
@@ -8,6 +8,10 @@
 #include "game/ui/TypeInfo.hh"
 #include "game/ui/UIControl.hh"
 
+namespace Sound {
+enum class SoundId;
+} // namespace Sound
+
```
Comment (stblr) [inline payload/game/ui/Page.hh:14]: I don't think that's needed.

Comment (riidefi) [reply payload/game/ui/Page.hh:14]: As in, please include `SoundId` in this header?

Comment (stblr) [reply payload/game/ui/Page.hh:14]: It's already included in `Page.hh`.

Comment (riidefi) [reply payload/game/ui/Page.hh:14]: You're right

**Review by stblr — APPROVED** [review]

**Pull request merged**